frontend/src/layout/header/Search.tsx

```diff
@@ -43,7 +43,6 @@ export const SearchField: React.FC<SearchFieldProps> = ({ variant }) => {
 
     const extraCss = variant === "desktop"
         ? {
-            maxWidth: 280,
```

Comment:
I noticed that it would't update the width when only using `maxWidth`. Even in the old version it would never actually be 280px. I did a quick google search, and apparently you also need to declare a `width: 100%` for this to work.